code/modules/awaymissions/mission_code/Cabin.dm

```diff
@@ -89,6 +90,9 @@
 	if(istype(T,/turf/open/floor/plating/asteroid/snow))
 		return ..(T)
 	return 0
+	
+/datum/mapGeneratorModule/bottomlayer/snow
+	spawnableAtoms = list(/turf/open/floor/plating/asteroid/snow/atmosphere = 100)
```

Comment:
Please don't use this list for turfs, use ```spawnableTurfs```, It will work, but just not quite as intended in some instances (cluster checks, etc.)